addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -365,12 +365,12 @@ <H2 id="id-10062">PII Disclosure</H2>
 <p>
 At MEDIUM and HIGH threshold, the following content types are evaluated:
 <ul>
-    <li>HTML</li>
+    <li>HTML (visible text and script blocks)</li>
     <li>JSON</li>
     <li>XML</li>
 </ul>
 Image and CSS files are always ignored.
-Every other content type is evaluated at LOW threshold.
+Every other content type is evaluated at LOW threshold. Additionally at LOW threshold the entire response is evaluated.
```

Comment:
Maybe this should be more explicit that's for HTML, since the others are always.